src/components/playerInput/index.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { useState } from 'react';
+
+import './style/player.css';
+
+export default function PlayerInput({ children }) {
+  return <form className="inputForm">{children}</form>;
+}
+
+PlayerInput.Input = function Input({ children, ...props }) {
```

Comment:
it is also possible, to extract the props directly in the function parameters:

```suggestion
PlayerInput.Input = function Input({ children, type, name, ...props }) {
```